conan/recipe/conanfile.py

```diff
@@ -45,59 +47,60 @@ class TargetConan(ConanFile):
         "nativium_code_coverage": False,
     }
     exports_sources = "*"
-    generators = "cmake"
+    generators = "CMakeToolchain", "CMakeDeps"
```

Comment:
Having a generator in two place has weird behavior,
Toolchain is in `generare()` so you dont want it here in the attribute

```suggestion
    generators = "CMakeDeps"
```

Reply:
Hi @prince-chrismc. Thanks for this. But after apply, i run again it does nothing different. The problem is same.